webpack/assets/javascripts/react_app/common/I18n/intl.js

```diff
@@ -0,0 +1,7 @@
+const [htmlElemnt] = document.getElementsByTagName('html');
```

Comment:
There is a typo in the var name though :)
```suggestion
const [htmlElement] = document.getElementsByTagName('html');
```